### PR TITLE
Remove unused scripts, unused script blocks, and add explicit "arches" files to remove "i386" from future Eoan updates

### PR DESCRIPTION
**Pull request by tianon**
See https://lists.ubuntu.com/archives/ubuntu-devel-announce/2019-June/001261.html

### CODE REVIEW

**Comment by mwhudson** [comment]
Although of course, as things turned out, lol

I guess this is still an improvement, even if we put i386 back for eoan?

**Comment by tianon** [comment]
Yeah for sure -- it's still better to be explicit IMO so we can consider things like dropping an arch (or maybe even adding another arch someday). :+1:

(And there's a lot of cruft here we don't need anymore that this is ditching.)

File: eoan/arches [hunk 1]
```diff
@@ -0,0 +1,5 @@
+amd64
+arm64
+armhf
+ppc64el
```
Comment (tianon) [inline eoan/arches]: So should we add `i386` back here?

Comment (mwhudson) [reply eoan/arches]: We still seem to be producing i386 images for eoan so yes, let's put it back for now.

**Comment by tianon** [comment]
Ok, updated :+1:

**Comment by mwhudson** [comment]
Still LGTM

**Pull request merged**